src/components/CopyToClipBoardIconOnly.tsx

```diff
@@ -9,24 +9,28 @@
  */
 
 import { toast } from '$app/common/helpers/toast/toast';
+import { MouseEvent } from 'react';
 import { MdOutlineContentCopy } from 'react-icons/md';
 
 interface Props {
   text: string;
   className?: string;
   secure?: boolean;
   limit?: number;
+  stopPropagation?: boolean;
 }
 
 export function CopyToClipboardIconOnly({
   text,
   className,
   secure,
   limit = 0,
+  stopPropagation,
 }: Props) {
   const value = text || '';
 
-  const handleClick = () => {
+  const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+    stopPropagation && event.stopPropagation();
```

Comment:
Can you guide me what this has to do with this PR? Thanks.

Reply:
@beganovich Actually, we open the slider when the user clicks on one of the first three columns, and this can be one of them, which is actually quite common. So, to prevent the slider from opening when the user clicks on the icon just to copy the number, I implemented stopPropagation here. This event's function will simply disable the parent's onClick event, which, in our case, is opening the slider. I believe that opening the slider at the moment of copying the number breaks the UX. Let me know your thoughts.